torch/_dynamo/variables/torch.py

```diff
@@ -397,36 +401,20 @@ def call_function(
                 )
             else:
                 unimplemented(f"torch.from_numpy(<{type(t)}>)")
-        elif len(args) > 0 and isinstance(args[0], TensorWithTFOverrideVariable):
-            # This code block implements inlining the __torch_function__
-            # override of a tensor.
-
-            tensor_with_tf_override = args[0]
-
-            # TODO(future PR): make this implement the full __torch_function__ API
-            # instead of assuming the relevant override is in the first argument.
-            args[0] = args[0].tensor_variable
-
-            unwrapped = TensorWithTFOverrideVariable.inline_torch_function_unwrapped(
-                tx,
-                self,
-                tensor_with_tf_override.orig_tensor_variable_source,
-                tensor_with_tf_override.subclass_torch_function__func,
-                tensor_with_tf_override.subclass_type,
-                options,
-                args,
-                kwargs,
-            )
+        elif can_dispatch_torch_function(tx, args, kwargs):
+            # continue the assumption that args[0] is the torch function
+            # tensor until we fully trace the base torch function impl
```

Comment:
sorry, is this a "TODO for upcoming PR in the stack?" Or a description of what the code does

Reply:
This is a todo for the next PR, this PR purely is refactoring TensorWithTFOverrideVariable without adding anything new.

Reply:
I'll be able to remove this whole branch and just call `call_torch_function` with the next PR